jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractScreenListPresenter.java

```diff
@@ -157,12 +163,27 @@ public void setupDetailBreadcrumb(PlaceManager placeManager,
         breadcrumbs.addBreadCrumb(getPerspectiveId(),
                                   listLabel,
                                   () -> closeDetails(detailScreenId));
+
         breadcrumbs.addBreadCrumb(getPerspectiveId(),
                                   detailLabel,
                                   Commands.DO_NOTHING);
         this.detailScreenId = detailScreenId;
     }
 
+    public void setupManageDetailBreadcrumb(PlaceManager placeManager,
+                                            String listLabel,
+                                            String detailLabel,
+                                            String detailScreenId) {
+        setupDetailBreadcrumb(placeManager,
+                              getManageListName(listLabel),
+                              detailLabel,
+                              detailScreenId);
+    }
+
+    private String getManageListName(String listName) {
+        return Constants.INSTANCE.Manage() + " " + listName;
```

Comment:
@nmirasch I'm not sure if concatenating literals will work when it comes to having the entire text localized in different languages. I think best would be to keep these as individual translation keys with the complete label: "Manage Process Instances" etc.